README.md

```diff
@@ -85,6 +92,20 @@ Sanity and unit tests are run as normal:
   ansible-test sanity
 ```
 
+Integration tests require real GCP credentials that must be provided to ansible-test. To run integration tests locally using service account credentials:
+
+1. [Create a service account key](https://support.google.com/cloud/answer/6158849?hl=en&ref_topic=6262490#serviceaccounts&zippy=%2Cservice-accounts) in the GCP console and save the json cred file.
+2. Create the file `tests/integration/cloud-config-gcp.ini` containing the following:
+
+   ```ini
+    [default]
+    gcp_project: <project ID>
+    gcp_cred_file: </path/to/cred/file.json>
+    gcp_cred_kind: serviceaccount
+   ```
+
+3. Run `ansible-test integration`
```

Comment:
We should also mention that there's also possible to run the integration tests on a specific target.

Reply:
Done